lib/matplotlib/widgets.py

```diff
@@ -879,8 +879,7 @@ def _click(self, event):
         if self.ignore(event):
             return
         if event.inaxes != self.ax:
-            self.capturekeystrokes = False
-            self.stop_typing()
+             self.stop_typing()
```

Comment:
Idonteven... thanks. 
I'm new to this whole contributing to public projects thing... I marked this PR as editable by the mpl powers that be, but I don't know that just means this thread or the committed code itself. Given that git is telling me that my remote is unchanged, I guess it means the former (your diff in which you show the fixed code notwithstanding), so I went ahead and pushed the typo fix myself. Is that right?